tests/supbot/test_vacancy.py

```diff
@@ -338,7 +359,41 @@ async def test_sending_resume(
             "reply_markup": SENDING_INFO_KEYBOARD_MARKUP,
         },
     )
+    mocked_bot.assert_no_more_api_calls()
+
+
+@pytest.mark.anyio()
+async def test_sending_resume_unsupported(
```

Comment:
suggestion: (для консистентности)
```suggestion
@pytest.mark.anyio()
async def test_sending_resume_unsupported_message(
```

Reply:
поправил